west.yml: Updated the hal silabs version

ragurram26 opened the pull request:
To ensure validation for changes in hal_silabs,
this commit updates west.yml to point to cb17629c3f00b6c547b6bc9f29863af33f8e1d65.

jhedberg commented:
@ragurram26 until https://github.com/zephyrproject-rtos/hal_silabs/pull/94 is merged you need to reference that PR instead of any specific commit. See https://docs.zephyrproject.org/latest/develop/modules.html#process-for-submitting-changes-to-existing-modules

ragurram26 commented:
@jhedberg I have updated my PR

jerome-pouiller commented:
@ragurram26 Compliance fails because you need to fix your name in your `gitconfig`. Once done, `git commit --amend --reset-author -s` should update the author of the commit.

ragurram26 committed:
west.yml: Updated the hal silabs version

To ensure validation for changes in hal_silabs,
this commit updates west.yml to point to cb17629c3f00b6c547b6bc9f29863af33f8e1d65

Signed-off-by: Rahul Gurram <rahul.gurram@silabs.com>

jhedberg commented:
@ragurram26 please remove the mention of "pull/94/head" in the commit message. That's not relevant since the HAL PR was already merged.

ragurram26 commented:
@jhedberg I have updated the commit message

jhedberg commented:
@ragurram26 there are a bunch of PRs currently open that update the hal_silabs reference. E.g. I just merged the sleeptimer PR in the HAL tree. I think it'd make sense to try to consolidate the main tree PRs - maybe close this one and let the changes be pulled in through the other open PRs that update hal_silabs in the manifest?

jerome-pouiller commented:
Superseded by https://github.com/zephyrproject-rtos/zephyr/pull/87606
